ui/.env.test

```diff
@@ -1,5 +1,29 @@
-COOKIE_SECRET="keyboardmash"
-MAGIC_LINK_SECRET="potatopotatopotato"
-DEPLOY_URL="localhost:3001"
-LANDING_PAGE_URL="https://cobudget.webflow.io"
-FROM_EMAIL="noreply@cobudget.com"
+COOKIE_SECRET="A_LONG_RANDOM_VALUE_THAT_IS_NOT_THIS_STRING"
+DATABASE_URL="postgres://postgres:postgres@localhost:35432/cobudget"
+POSTMARK_FROM_EMAIL="fake"
+POSTMARK_API_TOKEN="fake"
+MAGIC_LINK_SECRET="83uj*U8j3r!@3"
+FROM_EMAIL="test@test.com"
+FACEBOOK_APP_ID="656966848728529"
+FACEBOOK_APP_SECRET="9406673d6841ded46f6d269d920f4d14"
+
+#CROWDIN_PROJECT_ID=513056
+#CROWDIN_API_TOKEN=82141651739eb8ba3e75238aea058cc25f0b98ab36e5ff2b7e476ddd477bcf1437ded4432848ab98
+
+CROWDIN_PROJECT_ID="510498"
+CROWDIN_API_TOKEN="4107617345ef84b7f0d76a8dc0805bad0244a1ac4b69ce6b90e5099b321fa60f2dff0727384093d3"
+
+#STRIPE_MONTHLY_PLAN_PRICE_ID="price_1LDnRqFROlXOf2Bwmp0TOgFI"
+#STRIPE_YEARLY_PLAN_PRICE_ID="price_1LF34sFROlXOf2BwERlXlLh4"
+#STRIPE_API_KEY="sk_test_51EpKsJFROlXOf2BwOB48vzuamdT6Bd8EZQxb8BsTGgLGirYUSm8h8q3dZe55sFmBSJI8d4TD1UM0zVSrGwPZKrTr006z7P5Lva"
+##STRIPE_WEBHOOK_SECRET="whsec_7b26788818f8d75432ebbbe608493780d6fda2b917d453d0d3c4f847f41ef22a"
+#STRIPE_WEBHOOK_SECRET="whsec_b81b3757d08c8e6172685dbd071e0482bc89a2b93a31d9444c2027ea73d5ee7c"
+
+STRIPE_MONTHLY_PLAN_PRICE_ID="price_1Lkb2JJ43b4LSsbe5Qj2eftc"
+STRIPE_YEARLY_PLAN_PRICE_ID="price_1Lkb2JJ43b4LSsbexJdXrpYQ"
+STRIPE_API_KEY="sk_test_51LjmgRJ43b4LSsbeP24tSN0twDABOxn8BmT1835zlkh1Md6i6oSg2VVrTnIIR71fZKuijkqOFcXkwaC7UU5FeXxC005DkyveHh"
+STRIPE_ACCOUNT_WEBHOOK_SECRET="whsec_b81b3757d08c8e6172685dbd071e0482bc89a2b93a31d9444c2027ea73d5ee7c"
+STRIPE_CONNECT_WEBHOOK_SECRET="whsec_b81b3757d08c8e6172685dbd071e0482bc89a2b93a31d9444c2027ea73d5ee7c"
+STRIPE_WEBHOOK_SECRET="whsec_b81b3757d08c8e6172685dbd071e0482bc89a2b93a31d9444c2027ea73d5ee7c"
+
+ERROR_REPORTING_WEBHOOK="https://discord.com/api/webhooks/1025000522022920192/1xjiaJ86fl2EzhJoUOxNwOdW1kIeRM2Krgql1fKlmUH9WJ62zAt11QN4xdEoU93N_jzW"
```

Comment:
Are all of these needed for the tests and are we sure nothing in here is a secret?

Reply:
@alinauroz have a look